types/database/schema/ChatListDetail.types.ts

```diff
@@ -39,6 +39,7 @@ export interface User {
   online: boolean;
   name: string;
   image: string;
+  username: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `username` field in the interface seems fine. It doesn't seem to introduce any issues.
```diff
+   username: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->